frontend/src/components/main/main_page_container.js

```diff
@@ -1,12 +1,13 @@
 import { connect } from 'react-redux';
```

Comment:
remeber to put an empty line between module imports and imports from files we have written